frameworks/cockroachdb/src/main/dist/svc.yml

```diff
@@ -0,0 +1,196 @@
+name: {{FRAMEWORK_NAME}}
+scheduler:
+  principal: {{FRAMEWORK_PRINCIPAL}}
+pods:
+  cockroachdb:
+    count: {{NODE_COUNT}}
+    uris:
+      - {{COCKROACH_URI}}
+      - {{BOOTSTRAP_URI}}
+    {{#ENABLE_VIRTUAL_NETWORK}}
+    networks:
+      dcos:
+    {{/ENABLE_VIRTUAL_NETWORK}}
+    placement: {{NODE_PLACEMENT}}
+    resource-sets:
+      server-resources:
+        cpus: {{NODE_CPUS}}
+        memory: {{NODE_MEM}}
+        ports:
+          gui:
+            port: {{CONTAINER_GUI_PORT}}
+            vip:
+              prefix: gui
+              port: {{COCKROACH_GUI_PORT}}
+          internal:
+            port: {{CONTAINER_INTERNAL_PORT}}
+            vip:
+              prefix: internal
+              port: {{COCKROACH_INTERNAL_PORT}}
+        volume:
+          path: "cockroach-data"
+          type: {{NODE_DISK_TYPE}}
+          size: {{NODE_DISK}}
+      sidecar-resources:
+        cpus: {{SIDE_CPUS}}
+        memory: {{SIDE_MEM}}
+    tasks:
+      node-init:
```

Comment:
are we sure this will never be run more than once? if this command is run in a directory where a cluster previously ran, the old cluster is effectively re-initialized (read: destroyed)